app/src/main/java/com/riadh/movies/utils/MyUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.riadh.movies.utils;
+
+
+import android.content.Context;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+
+import com.riadh.movies.app.MyApplication;
+import com.riadh.movies.models.Genre;
+
+import java.util.List;
+
+public class MyUtils {
+
+    public static String getGenresString(List<Integer> genreIds, MyApplication app) {
+        StringBuilder res = new StringBuilder();
+
+        for (Genre g : app.getGenres().getGenres()) {
+            for (Integer id : genreIds) {
+                if (id.equals(g.getId())) {
+                    res.append(g.getName()).append(", ");
+                }
+            }
+        }
```

Comment:
Here, your code is taking certain amount of time. We can do some simple analysis here:
```
    StringBuilder res = new StringBuilder();
    Long initialTime = System.nanoTime();
    for (Genre g : app.getGenres().getGenres()) {
      for (Integer id : genreIds) {
        if (id.equals(g.getId())) {
          res.append(g.getName()).append(", ");
        }
      }
    }
    Log.d("TIME", "1) spent time: " + (System.nanoTime() - initialTime));
```
What do you think of, instead of using a `for` * `for`, doing a little change in your `MyApplication` to store your `Genres` like `HashMaps`. If you do so, you could use a code like this here:
```
    // MyApplication
    Map<Integer, String> genresMap = new HashMap<>();
    for (Genre g : app.getGenres().getGenres()) {
      genresMap.put(g.getId(), g.getName());
    }

    // MyUtils
    Long initialTime = System.nanoTime();
    StringBuilder res = new StringBuilder();
    for (Integer id : genreIds) {
      if (genresMap.containsKey(id)) {
        res.append(genresMap.get(id)).append(", ");
      }
    }
    Log.d("TIME", "2) spent time: " + (System.nanoTime() - initialTime));
```
Is it worth thinking that way? Why?